docs/4.3/enterprise/workflow/ssh_approval_jira_server.md

```diff
@@ -58,47 +68,48 @@ The above sequence should result in three PEM encoded files being generated: aut
-Teleport uses this field to reference it's internal request ID. If anyone changes this field on Jira, or tries to forge the permission request, Teleport will validate it and ignore it.
+Teleport stores the request metadata in a special Jira custom field that must be named teleportAccessRequestId. To create that field, go to Administration -> Issues -> Custom Fields -> Add Custom Field.
 
-#### Getting your Jira API token
-If you're using Jira Cloud, navigate to [Account Settings -> Security -> API Tokens](https://id.atlassian.com/manage/api-tokens) and create a new app specific API token in your Jira installation.
-You'll need this token later to configure the plugin.
+Name the field teleportAccessRequestId, and choose Text Field (single line) as the field type. Assign the field to your project, or make it global. Teleport Access Request ID is an internal field and it's not supposed to be edited by users, so you can leave the Screens section blank. That means that the field won't show up in Jira UI.
 
-For Jira Server, the URL of the API tokens page will be different depending on your installation.
+Go to Project Settings -> Fields and make sure that the teleportAccessRequestId field shows up on the list of fields available in this project.
 
+#### Setting up the status board
+The default Jira Software workflow has a different board setup from what Teleport needs, so we'll setup another workflow and assign that workflow to the project board.
+
+Go to Administration -> Workflows. You can choose to add a new workflow (recommended), or edit the existing workflow, it'll be called Software Simplified Workflow for Project NAME by default. It's only used in your single project, so it's safe to edit it.
+
+Edit the workflow to have these three states:
+
+1. Pending
+2. Approved
+3. Denied
+The rules of the workflow must meet these requirements:
 
-#### Settings up Jira Webhooks
-Go to Settings -> General -> System -> Webhooks and create a new Webhook for Jira to tell the Teleport Plugin about updates.
+- New created issues should be in Pending state.
+- It should be possible to move from Pending to Approved
+- It should be possible to move from Pending to Declined.
+- You can choose to make the workflow strict and restrict moving requests from Approved state to Declined state and vice versa, or leave that flexible. Teleport will only change the request status once, i.e. the first time the request is approved or denied on your Jira board.
 
-For the webhook URL, use the URL that you'll run the plugin on. It needs to be a publicly accessible URL that we'll later setup.
-Jira requires the webhook listener to run over HTTPS.
+With Workflow editor you can setup who can approve or deny the request based on their Jira user permissions. We won't cover that in this guide as it mostly relates to Jira settings and Teleport will by default allow anyone who can use the workflow to approve or deny the request.
```

Comment:
Jira settings and Teleport will by -> Jira settings. Teleport will by (add full stop - the statement about Teleport allowing anyone who can use the workflow to approve/deny is important)